app/views/student_review/_responses.html.erb

```diff
@@ -59,6 +64,7 @@
                         <td>
                           <% if (!@latest_response.is_submitted) %>
                               <%= link_to "Edit", {:controller => 'response', :action => 'edit', :id => @latest_response.id} %>
+                	      <%= link_to "Drop", {:controller => 'review_mapping', :action => 'drop_review', :id => @participant.id ,:map_id => map.id} %>
```

Comment:
This requirement was not in our project scope (Requirement 2: https://docs.google.com/document/d/1fI5KbPXCJ8dSUyLynnqC__A3MtZ47enNfMcki3Vf3uk/edit). But we can add this functionality to avoid feature misuse as indicated by professor during demo.